app/assets/stylesheets/govuk-component/govspeak/_steps.scss

```diff
@@ -1,3 +1,6 @@
+$black: #000;
+$white:#fff;
```

Comment:
Black and white are already defined here:
https://github.com/alphagov/govuk_frontend_toolkit/blob/89168934d1455754f455c398ba5c350a484d9a2a/stylesheets/colours/_palette.scss